data/taylor_series/taylor_series.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+import math
+
+def func_cos(x, n):
+    cos_approx = 0
+    for i in range(n):
+        coef = (-1)**i
+        num = x**(2*i)
+        denom = math.factorial(2*i)
+        cos_approx += (coef) * ((num)/(denom))
+    return cos_approx
+
+x = math.radians(45)
+n = 8
+
+result = True if np.isclose(taylor_series(func_cos, x, n), np.cos(x)) else False
```

Comment:
Should'nt the result be `np.isclose(taylor_series(np.cos, x, n), func_cos(x))` ? I am assuming you want to compare the output from your function (which computes the taylor series approximation) against the actual taylor series approximation you have computed for the cos function.

Reply:
I agree with Nav, maybe should be `np.isclose(taylor_series(np.cos, x, n), func_cos(x, n))`